layouts/_shortcodes/handbook-data-toc.html

```
@@ -0,0 +1,36 @@
+	</ul>
+{{ end }}
+
+
+{{ define "sublinks" }}
+<ul>
+	{{ $page := .page}}
+	{{ range .links }}
+	{{ $u := urls.Parse .url }}
+	{{ $url := .url }}
+	{{ if not $u.IsAbs }}
+		<li><a href="{{ relref $page .url }}">{{.name}}</a></a></li>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </a> ]

(tag-pair)